fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -967,12 +973,17 @@ public void refreshMaterializedView(MvTaskRunContext mvContext, ExecPlan execPla
             StmtExecutor parentStmtExecutor = ctx.getParent().getExecutor();
             parentStmtExecutor.registerSubStmtExecutor(executor);
         }
-        ctx.setStmtId(new AtomicInteger().incrementAndGet());
+        ctx.setStmtId(STMT_ID_GENERATOR.incrementAndGet());
         ctx.setExecutionId(UUIDUtil.toTUniqueId(ctx.getQueryId()));
+        LOG.info("start to refresh materialized view {}, query_id:{}", materializedView.getName(),
+                ctx.getQueryId());
         try {
             executor.handleDMLStmt(execPlan, insertStmt);
         } finally {
+            LOG.info("finished to refresh materialized view {}, query_id:{}", materializedView.getName(),
+                    ctx.getQueryId());
             QeProcessorImpl.INSTANCE.unregisterQuery(ctx.getExecutionId());
+            GlobalStateMgr.getCurrentState().getMetadataMgr().removeQueryMetadata();
```

Comment:
Looks like do not need this

Reply:
just to ensure to remove query meta cache?